File: ZooJava/src/main/java/com/example/zoojava/repositories/employees/EmployeesRepository.java
package com.example.zoojava.repositories.employees;

import com.example.zoojava.models.Employee;
import com.example.zoojava.repositories.ICRUD;
import javafx.collections.ObservableList;


public interface EmployeesRepository extends ICRUD<Employee> {
    ObservableList<Employee> findAll();
    Employee findByEmail(String email);
}

File: ZooKotlin/src/main/kotlin/com/example/zookotlin/repositories/employees/EmployeesRepository.kt
package com.example.zookotlin.repositories.employees

import com.example.zookotlin.models.Employee
import com.example.zookotlin.repositories.ICRUD
import javafx.collections.ObservableList

interface EmployeesRepository: ICRUD<Employee> {
    fun findAll(): ObservableList<Employee>
    fun findByEmail(email:String): Employee?
}